### ISSUE TITLE
Is there any optimization option when running c# codes?

### ISSUE THREAD

**Issue by faruknane** [dotnet/try#756]
Are we enabled to run C# codes with an optimized way in jupyter notebook? There is a property named "Optimize Code" in Visual Studio to let the compiler knows whether the code will be optimized or not. Something like this do you provide us?

I'm asking this feature because there is x10 (for loop only small benchmarking) performance gap between Try .Net and .Net Core (Optimize Code is enabled), and x2 performance gap when Optimize Code is disabled.

**Comment by faruknane** [comment]
I hope I didn't ask something wrong

**Comment by jonsequitur** [comment]
Not at all, it's a good question. We're not looking deeply into performance optimization at this stage, but we will as we stabilize the code. We do have support for loading native libraries from NuGet packages, if that's helpful.

**Comment by faruknane** [comment]
I will look into that, thanks. Anyway with the current state, it's probably far better than python.
